web/src/Root.tsx

```diff
@@ -345,11 +348,10 @@ const Root = () => {
   };
 
   const onRootMounted = () => {
-    fetchKotsAppMetadata();
-    ping();
-    checkIsHelmManaged();
-
     if (Utilities.isLoggedIn()) {
+      fetchKotsAppMetadata();
+      ping();
+      checkIsHelmManaged();
```

Comment:
I believe all of these will still be called even if they're bypassed here.

metadata is refetched by `AppDetails`
ping is run in an interval
isHelmManaged is called after successful login in the `SecureAdmin` component